src/Illuminate/Database/PDO/Concerns/ConnectsToDatabase.php

```diff
@@ -13,7 +13,7 @@ trait ConnectsToDatabase
      * @param  array  $params
      * @return \Illuminate\Database\PDO\Connection
      */
-    public function connect(array $params)
+    public function connect(array $params, $username = null, $password = null, array $driverOptions = [])
```

Comment:
This is a BC break. Also, missing phpdoc.

Reply:
thanks for pointing out. I will fix the code up once we decided on what the desired way of solving this is. Please see my comment.